src/Stratis.Bitcoin.Features.Consensus/CoinViews/Coindb/FasterCoindb.cs

```diff
@@ -333,6 +333,41 @@ public void SaveChanges(IList<UnspentOutput> unspentOutputs, HashHeightPair oldB
             this.performanceCounter.AddInsertedEntities(insertedEntities);
         }
 
+        /// <inheritdoc />
+        public int GetMinRewindHeight()
```

Comment:
Gustav, we don't actually use FasterDb at all, so we can just actually delete this ;)

Reply:
We need the method to avoid a compiler error so let's just leave that for now.

Reply:
Sorry I meant delete the whole file ;)

Reply:
Done.